File: springboot/open-api/src/main/java/com/openapi/domain/constant/ws/WsPushTarget.java
package com.openapi.domain.constant.ws;

import lombok.Getter;

/**
 * 服务端下行 {@link WsEvent#CONTROL_COMMAND_SB} 等消息时的推送目标。
 */
public enum WsPushTarget {

    /** 发往 Android App 客户端 */
    ANDROID("android"),

    /** 发往 RK 设备客户端 */
    RK("rk");

    @Getter
    private final String value;

    WsPushTarget(String value) {
        this.value = value;
    }
}

File: app/android/app/src/main/java/com/magicvector/domain/constant/ws/WsPushTarget.kt
package com.magicvector.domain.constant.ws

/**
 * 服务端下行 [WsEvent.CONTROL_COMMAND_SB] 等消息时的推送目标。
 */
enum class WsPushTarget(val value: String) {
    /** 发往 Android App 客户端 */
    ANDROID("android"),

    /** 发往 RK 设备客户端 */
    RK("rk")
}
